packages/dep-check/test/capabilities.test.ts

```diff
@@ -86,14 +87,14 @@ describe("resolveCapabilities()", () => {
 
   test("dedupes packages", () => {
     const packages = resolveCapabilities(
-      ["core-android", "core-ios", "test-app"],
+      ["core", "core", "test-app"],
```

Comment:
```suggestion
      ["core", "test-app"],
```

do you need "core" twice here?

Reply:
Yes, this tests that `core` is deduped. `core`, `core-android` and `core-ios` all point to the same package, but I had in mind to change the latter two to be meta.